.changelog/3498.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+resource/cloudflare_access_mutual_tls_certificate: sort associated hostnames alphabetically
```

Comment:
Thanks for the fix! Just a nit that this changelog still says the hostnames are being sorted alphabetically, which isn't the case anymore.

Reply:
Nice catch! Fixed that :)